src/main/scala/io/iohk/ethereum/rlp/package.scala

```diff
@@ -20,10 +20,14 @@ package object rlp {
 
   def encode[T](input: T)(implicit enc: RLPEncoder[T]): Array[Byte] = RLP.encode(enc.encode(input))
 
+  def encodeToRlp[T](input: T)(implicit enc: RLPEncoder[T]): RLPEncodeable = enc.encode(input)
+
   def encode(input: RLPEncodeable): Array[Byte] = RLP.encode(input)
 
   def decode[T](data: Array[Byte])(implicit dec: RLPDecoder[T]): T = dec.decode(RLP.rawDecode(data))
 
+  def decode[T](data: RLPEncodeable)(implicit dec: RLPDecoder[T]): T = dec.decode(data)
```

Comment:
I think that this function and the `encodeToRLP` function might get confused with the `encode` and `decode` that use the `RLP` internal versions. Maybe they should be renamed to something that more explicitly differentiates them.